### PR TITLE
[docs][yba] yb controller and versions supported

### PR DESCRIPTION
**Pull request by ddhodge**
Document that YBC is only compatible with DB clusters that are 2.16 and later

@netlify /preview/yugabyte-platform/back-up-restore-universes/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/_index.md [hunk 1]
```diff
@@ -13,7 +13,16 @@ weight: 645
...
+- On-demand [backup](back-up-universe-data/) and [restore](restore-universe-data/).
+- [Scheduled backups](schedule-data-backups/). Schedule backups at regular intervals, along with retention periods.
+- [Incremental backups](back-up-universe-data/ysql/#create-incremental-backups). Create a schedule to take full backups periodically and incremental backups between those full backups.
+- [Configurable performance parameters](back-up-universe-data/ysql/#configure-backup-performance-parameters). Tune parallelization and buffers for faster backup and restore performance. In most cases, this results in 5x or more speed improvements in backups and restores.
+
+{{< note title="Note" >}}
+Configurable performance parameters and incremental backups are mediated using the yb-controller process, which is only available in YBA 2.16 or later for universes with database version 2.16 or later.
+{{< /note >}}
 
 <div class="row">
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/_index.md:27]: restructure the tiles based on its left nav order.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md [hunk 2]
```diff
@@ -34,7 +34,11 @@ type: docs
 
 </ul>
 
-<br>You can use YugabyteDB Anywhere to back up your YugabyteDB universe YSQL data.
+You can use YugabyteDB Anywhere to back up your YugabyteDB universe YSQL data.
+
+If you are using YBA version 2.16 or later to manage universes using database version 2.16 or later, in addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can add an [incremental backup](#create-incremental-backups) and [configure throttle parameteers](#configure-throttle-parameters).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md:39]:
```suggestion
If you are using YBA version 2.16 or later to manage universes using database version 2.16 or later, in addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can add an [incremental backup](#create-incremental-backups) and [configure throttle parameters](#configure-throttle-parameters).
```

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md [hunk 3]
```diff
@@ -29,47 +29,51 @@ type: docs
...
 </ul>
 
-<br>You can use YugabyteDB Anywhere to back up your YugabyteDB universe YCQL data.
+You can use YugabyteDB Anywhere to back up your YugabyteDB universe YCQL data.
+
+For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql/).
 
 Note that non-transactional backups are not supported.
 
+If you are using YBA version 2.16 or later to manage universes using database version 2.16 or later, in addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can add an [incremental backup](../ysql/#create-incremental-backups) and [configure performance parameteers](../ysql/#configure-backup-performance-parameters).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md:38]:
```suggestion
If you are using YBA version 2.16 or later to manage universes using database version 2.16 or later, in addition to actions such as deleting and restoring the backup, as well as restoring and copying the database location, you can add an [incremental backup](../ysql/#create-incremental-backups) and [configure performance parameters](../ysql/#configure-backup-performance-parameters).
```

**Review by aishwarya24 — COMMENTED** [review]
some comments at a quick glance.

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups/ysql.md [hunk 4]
```diff
@@ -63,15 +63,15 @@ Before scheduling a backup of your universe YSQL data, create a policy, as follo
 
 1. Specify the interval between backups or select **Use cron expression (UTC)**.
 
-1. For a YB Controller-powered universe, you can enable **Take incremental backups within full backup intervals** to instruct the schedule policy to take full backups periodically and incremental backups between those full backups. The incremental backups intervals be longer than the full scheduled backup frequency:
+1. Enable **Take incremental backups within full backup intervals** to instruct the schedule policy to take full backups periodically and incremental backups between those full backups (YBA version 2.16 or later, and YugabyteDB version 2.16 or layer only). The incremental backups intervals must be shorter than the full scheduled backup frequency:
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/schedule-data-backups/ysql.md:66]:
```suggestion
1. Enable **Take incremental backups within full backup intervals** to instruct the schedule policy to take full backups periodically and incremental backups between those full backups (YBA version 2.16 or later, and YugabyteDB version 2.16 or later only). The incremental backups intervals must be shorter than the full scheduled backup frequency:
```

File: docs/content/preview/yugabyte-platform/back-up-restore-universes/_index.md [hunk 5]
```diff
@@ -13,7 +13,17 @@ weight: 645
...
+You can use YugabyteDB to schedule and manage backups of your universe data. This includes the following features:
+
+- On-demand [backup](back-up-universe-data/) and [restore](restore-universe-data/).
+- [Scheduled backups](schedule-data-backups/). Schedule backups at regular intervals, along with retention periods.
+- [Incremental backups](back-up-universe-data/ysql/#create-incremental-backups). Create a schedule to take full backups periodically and incremental backups between those full backups.
+- [Configurable performance parameters](back-up-universe-data/ysql/#configure-backup-performance-parameters). Tune parallelization and buffers for faster backup and restore performance. In most cases, this results in 5x or more speed improvements in backups and restores.
+- [Point-in-time recovery](pitr/). Recover universe data from a specific point in time.
+
+{{< note title="Note" >}}
+Configurable performance parameters and incremental backups are mediated using the yb-controller process, which is only available in YBA 2.16 or later for universes with database version 2.16 or later.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/back-up-restore-universes/_index.md:25]: I couldn't find any doc about what is yb-controller to possibly link here. I am assuming thats coming up sometime soon?

Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/back-up-restore-universes/_index.md:25]: Nothing at the moment. Possibly.

**Pull request merged**